### PR TITLE
Todo List with React

### PR DESCRIPTION
**Pull request by umair-rao**
In this pull request, I built a single-page website with React having the following features. 
 - You can add your daily tasks.
 - You can remove the tasks by selecting and clicking on the delete button.

Please review my PR. 
Thanks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,239 @@
...
+<!-- PROJECT DESCRIPTION -->
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+# 📖 [Todo List] <a name="about-project"></a>
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+> Describe your project in 1 or 2 sentences.
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+**[Math Magicians]** is single page website which will allows you to perform simple calculation just like calculator. Website is react based.
```
Comment (Victoria129) [inline README.md:66]: - Kindly remove any lines that do not relate to the referenced project.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,239 @@
...
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+After you're finished please remove all the comments and instructions!
```
Comment (Victoria129) [inline README.md:26]: - Please remove any comments and instructions as the _instruction_ says.

File: README.md [hunk 3]
```diff
@@ -1,70 +1,239 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
```
Comment (Victoria129) [inline README.md:161]: - Please change Author1 to your name.

**Review by Victoria129 — COMMENTED** [review]
## Hi @umair-rao 🙋‍♀️

Good job on the project so far! There are a few changes required for your project to be complete.
Please consider the above in-line comments.

Cheers and Happy coding!

**Review by Victoria129 — CHANGES_REQUESTED** [review]
## Hi @umair-rao 🙋‍♀️

Good job so far!
There are some issues that you still need to work on to complete your project, but you are almost there!

## Highlights

✔️ No linter errors
✔️ Gitflow was correctly used


# Required Changes ♻️
 Please check the above in-line comments that I have included in the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

File: README.md (hunk 1 above)
Comment (umair-rao) [reply README.md:66]: Thanks for the correction all the suggested changes have been made.

File: README.md (hunk 3 above)
Comment (umair-rao) [reply README.md:161]: Thanks for the correction all the suggested changes have been made.

Comment (umair-rao) [reply README.md:161]: Thanks for the correction all the suggested changes have been made.

**Comment by umair-rao** [comment]
All the suggested changes have been made.
Thanks for your valuable suggestion.

**Pull request merged**